doc/Development-Guide-Rad-Tool.md

```diff
@@ -3,43 +3,46 @@
 
 ### Download And Install
 
- If your project version is 5.1.0+, all you have to do is just installing **ASP.NET Zero Power Tools** extension on visual studio, from https://marketplace.visualstudio.com/items?itemName=Volosoft.AspNetZeroPowerTools.
+ If your project is version 5.1.0+, all you have to do is just instal the **ASP.NET Zero Power Tools** extension on Visual Studio from https://marketplace.visualstudio.com/items?itemName=Volosoft.AspNetZeroPowerTools.
 
- If your project version is below 5.1.0, you also have to copy the AspNetZeroRadTool folder to your own project, from a newly downloaded 5.1.0+ project.
+ If your project is below version 5.1.0, you have to copy the AspNetZeroRadTool folder to your own project from a newly downloaded 5.1.0+ project.
 
- Tool doesn't support versions before v5.0.0.
+ This tool doesn't support versions before v5.0.0.
  
 ### How To Use It?
 
- Extension is inside **Tools** menu (Tools -> Asp.Net Zero -> Create An Entity). When you run it, you will see the interface for creating an entity. After carefully filling the fields, press **Generate** button to start the code generation process. 
+ The extension can be found inside the **Tools** menu (Tools -> Asp.Net Zero -> Create An Entity). When you run it, you will see the interface for creating an entity. After carefully filling out the fields, press the **Generate** button to start the code generation process. 
 
  <img src="images/RadToolUI.jpg" alt="Extension UI" class="img-thumbnail" width="507" height="440" />
 
- A simple console will appear and give you information about the process. If there is no warning and fail, run your project to see the results. You probably won't see a new page, but don't worry and grant yourself the required **permissions**.
+ A simple console will appear and give you information about the process. If there is no warning or fail, run your project to see the results. You probably won't see a new page because you must first grant yourself the required **permissions** in the application.
 
-Warning: If you are working on ASP.NET Core & Angular template, after generating the entity via Power Tools, run your ***.Web.Host** project and then run "**./angular/nswag/refresh.bat**" to update **service-proxies.ts**.
+Warning: If you are working on ASP.NET Core & Angular template, after generating the entity via Power Tools run your ***.Web.Host** project and then run "**./angular/nswag/refresh.bat**" to update **service-proxies.ts**.
 
- Warning: Be sure that you have saved your works before running this tool, since it will add new files and modify some of the existing files. We strongly recommend using a source control system (like Git). Otherwise, backup your project.
+ Warning: Be sure that you have saved your work before running this tool since it will add new files and modify some of the existing files. We strongly recommend using a source control system (like Git)-- otherwise, backup your project.
```

Comment:
The change from `. Otherwise` to `-- otherwise` is not appropriate.

Reply:
I'll make this change so we don't spend a lot of time on this minor topic, but you shouldn't start a sentence with 'Otherwise'.   The em-dash is used like a comma.  So, I feel the original is correct-- but I'll change it anyways.  https://www.quora.com/Is-it-correct-to-use-otherwise-in-a-beginning-of-a-sentence